<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyPhotosDialog</class>
 <widget class="QDialog" name="CopyPhotosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAidFrom">
       <property name="text">
        <string>139138529</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbFromGroup">
       <property name="text">
        <string>Group? (or user id)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGidFrom">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>13577027</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAidTo">
       <property name="text">
        <string>141847444</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbToGroup">
       <property name="text">
        <string>Group?</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGidTo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>29842458</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbWithComments">
     <property name="text">
      <string>With comments</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pteLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbToGroup</sender>
   <signal>clicked(bool)</signal>
   <receiver>leGidTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
